lib/grpperm.gi

```diff
@@ -736,6 +736,8 @@ BindGlobal("DoClosurePrmGp",function( G, gens, options )
             fi;
         od;
     else
+        o:=ValueOption("knownClosureSize");
```

Comment:
What do you intend for the `knownClosureSize` value option: if you're not using it in the code in the rest of the PR, and it's not documented, is there any point in it being there?